docs/framework/data/adonet/sql/windows-applications-using-callbacks.md

```diff
@@ -352,7 +349,7 @@ private void HandleCallback(IAsyncResult result)
         // You can create the delegate instance as you   
         // invoke it, like this:  
         this.Invoke(new DisplayInfoDelegate(DisplayStatus),  
-        String.Format("Ready(last error: {0}", ex.Message));  
+            $"Ready(last error: {ex.Message}");
```

Comment:
@mikkelbu nit not related to your changes: it should be a white space after "Ready"

Reply:
@pkulikov I've added a space here, and also in a similar case in the same file.